web_src/js/index.js

```diff
@@ -19,6 +19,7 @@ import initTableSort from './features/tablesort.js';
 import ActivityTopAuthors from './components/ActivityTopAuthors.vue';
 import {initNotificationsTable, initNotificationCount} from './features/notification.js';
 import {createCodeEditor} from './features/codeeditor.js';
+import {svg} from './svg.js';
```

Comment:
Are you sure this is necessary?

Reply:
No, it's a leftover from a previous attempt.